JS-Algorithms/challenges.js

```diff
@@ -13,7 +13,8 @@ Invoking "readableTime(3690)" should return "01:01:30" (HH:MM:SS)
 ***** */
 
 const readableTime = (seconds) => {
-  // YOUR CODE HERE...
+   if (seconds > 0) return new Date(seconds * 1000).toISOString().substr(11, 8);
+   return "Invalid seconds amount";
```

Comment:
`return seconds > 0 ? new Date(seconds * 1000).toISOString().substr(11, 8) : "Invalid seconds amount"`

Reply:
Thank you so much I would consider use ternary operators to improve my coding